libs/island-ui/core/src/lib/ToggleSwitch/ToggleSwitchLink.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { MouseEvent, useEffect, useState } from 'react'
 import { Link } from '../Link/Link'
+import { LinkProps } from 'next/link'
```

Comment:
Hmmm... yet  the ui-core `Link` component uses this exact same type  internally.

(See: https://github.com/island-is/island.is/blob/main/libs/island-ui/core/src/lib/Link/Link.tsx#L2)

Thus I feel no shame. :-)

Reply:
...I would not have dared to do this otherwise.

Could we decide to classify this issue as separate from this PR — something to resolve further up the food-chain at a later date?

Reply:
Oh well.. it's even returning a Next component, that's odd. Do you mind creating an issue so someone can take a look a it someday?

Reply:
sure.

Reply:
Feeling uninspired, I opened this embarrassingly terse issue: https://github.com/island-is/island.is/issues/4680